.travis.yml

```diff
@@ -1,17 +1,21 @@
 language: rust
+rust: nightly
```

Comment:
On 1.0.0, you can't enable unstable features afaik, and zinc makes pretty heavy use of some of them. Most notably, compiler plugins.